libkbfs/.gitignore

```diff
@@ -0,0 +1 @@
+libkbfs.test
```

Comment:
maybe generalize this and put `*.test` in the root `.gitignore`, which I think should apply to subdirs